src/test/testKit/addCommentDriver.ts

```diff
@@ -0,0 +1,26 @@
+import type { Locator, Page } from 'playwright';
+import { COMMENTS, COMMENT_INPUT, POST_COMMENT_BUTTON } from './selectors';
+
+export class AddCommentDriver {
```

Comment:
A see you already have a coments driver,

try to keep your drivers scoped to components rather than functionality.
I.e this should be the post-action-bar-driver.
in your ix tests, you will should use this driver to send a comment. while you use the commentsList  driver to check if the list was updated properly